io.github.lpub3d.LPub3D.json

```diff
@@ -0,0 +1,172 @@
+      "config-opts": [
+        "CONFIG+=3RD_PARTY_INSTALL=/app/lpub3d_linux_3rdparty",
+        "CONFIG+=release",
+        "CONFIG+=BUILD_CHECK"
+      ],
+      "sources": [
+        {
+          "type": "git",
+          "url": "https://github.com/trevorsandy/ldglite.git",
+          "tag": "v1.3.6",
+          "commit": "140865cb85b17eac1abe613066c9364db1e57650"
+        }
+      ]
+    },
+    {
+      "name": "ldview",
+      "buildsystem": "qmake",
+      "config-opts": [
+        "CONFIG+=3RD_PARTY_INSTALL=/app/lpub3d_linux_3rdparty",
+        "CONFIG+=release",
+        "CONFIG+=BUILD_CUI_ONLY",
+        "CONFIG+=USE_SYSTEM_LIBS",
+        "CONFIG+=BUILD_TINYXML",
+        "CONFIG+=BUILD_GL2PS",
+        "CONFIG+=BUILD_CHECK"
+      ],
+      "sources": [
+        {
+          "type": "git",
+          "url": "https://github.com/trevorsandy/ldview.git",
+          "branch": "qmake-build",
+          "commit": "16b44f3658690294b1faf13778e3654d443e3751"
+        }
+      ]
+    },
+    {
+      "name": "povray",
+      "buildsystem": "simple",
+      "build-commands": [
+        "cd unix && ./prebuild3rdparty.sh",
+        "./configure COMPILED_BY=\"Trevor SANDY <trevor.sandy@gmail.com> using FlatHub for LPub3D.\" --prefix=/app/lpub3d_linux_3rdparty LPUB3D_3RD_PARTY=yes --with-libsdl2 --with-boost-libdir=${FLATPAK_DEST}/lib --enable-watch-cursor",
+        "make",
+        "make install"
+      ],
+      "sources": [
+        {
+          "type": "git",
+          "url": "https://github.com/trevorsandy/povray.git",
+          "branch": "lpub3d/raytracer-cui",
+          "commit": "54b65529090b4161193df24f0d846046b1495f0b"
+        }
+      ],
+      "modules": [
+        {
+          "name": "boost",
+          "buildsystem": "simple",
+          "cleanup": [
+            "/include",
+            "/lib/libboost_*.a"
+          ],
+          "build-commands": [
+            "./bootstrap.sh --prefix=${FLATPAK_DEST} --with-libraries=date_time,thread",
+            "./b2 install variant=release link=shared runtime-link=shared --prefix=/app -j $FLATPAK_BUILDER_N_JOBS"
+          ],
+          "sources": [
+            {
+              "type": "archive",
+              "url": "https://downloads.sourceforge.net/project/boost/boost/1.60.0/boost_1_60_0.tar.bz2",
+              "sha256": "686affff989ac2488f79a97b9479efb9f2abae035b5ed4d8226de6857933fd3b"
+            }
+          ]
+        }
+      ]
+    },
+    {
+      "name": "lpub3d",
+      "buildsystem": "qmake",
+      "config-opts": [
+        "-makefile",
+        "-nocache",
+        "QMAKE_STRIP=:",
+        "CONFIG+=release",
+        "CONFIG+=flp",
+        "CONFIG+=build_check",
+        "CONFIG-=debug_and_release",
+        "INSTALL_PREFIX=/app"
+      ],
+      "sources": [
+        {
+          "type": "git",
+          "url": "https://github.com/trevorsandy/lpub3d-ci.git",
+          "tag": "v2.4.3",
+          "commit": "76d9418a484497f822a7044670101b7749ce7bb8"
```

Comment:
this is not the commit I see for tag `v2.4.3`

Reply:
Indeed. Not sure what happene here ?